recipes/straditize/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "straditize" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "2b62c05e70fa4b7e57ed8fc3e2205f10d5383a6000442dce5b4360fd22265ccd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install . --no-deps --ignore-installed -vvv
+  skip: true  # [py27]
```

Comment:
This selector should be py == 27

Reply:
sorry, resolved by https://github.com/conda-forge/staged-recipes/pull/7658/commits/16254977c51159ac7a87c247666ed6ba5e903412